worktree_test.go

```diff
@@ -1344,7 +1344,7 @@ func (s *WorktreeSuite) TestResetSparsely() {
 
 	sparseResetDirs := []string{"php"}
 
-	err := w.ResetSparsely(&ResetOptions{Mode: HardReset}, sparseResetDirs)
+	err := w.Reset(&ResetOptions{Mode: HardReset, SparseDirs: sparseResetDirs})
```

Comment:
Agree with the change, but let's do that straight into the v6 instead. So please rebase and change the PR target to the `v6-transport` branch.